core/src/main/java/org/geysermc/geyser/translator/protocol/java/level/JavaLevelChunkWithLightTranslator.java

```diff
@@ -79,9 +83,12 @@
 
 @Translator(packet = ClientboundLevelChunkWithLightPacket.class)
 public class JavaLevelChunkWithLightTranslator extends PacketTranslator<ClientboundLevelChunkWithLightPacket> {
+    private static final ThreadLocal<ExtendedCollisionsStorage> EXTENDED_COLLISIONS_STORAGE = ThreadLocal.withInitial(ExtendedCollisionsStorage::new);
 
     @Override
     public void translate(GeyserSession session, ClientboundLevelChunkWithLightPacket packet) {
+        final boolean USE_EXTENDED_COLLISIONS = !session.getBlockMappings().getExtendedCollisionBoxes().isEmpty();
```

Comment:
`useExtendedCollisions`

This case used is only for static constants.

Reply:
https://github.com/Kas-tle/Geyser/commit/ac249b0ebcedbc3e5be76b7a02684741a093fba5